src/plugins/dashboard/public/application/top_nav/dashboard_top_nav.tsx

```diff
@@ -517,7 +517,12 @@ export function DashboardTopNav({
             {
               'data-test-subj': 'dashboardUnsavedChangesBadge',
               badgeText: unsavedChangesBadge.getUnsavedChangedBadgeText(),
-              color: 'success',
+              title: '',
```

Comment:
Is this title field necessary?

Reply:
@ThomThomson ,
Yes, it is necessary as the default tooltip will be shown if `title` is not set to blank.

Reply:
Makes sense! Now we just need to get a review from the App Services team. I have reached out to them on slack